### PR TITLE
adding upgrade of SSL in provision.

### PR DESCRIPTION
**Pull request by shawn-hurley**
* Copy new ssl.conf for httpd
* generate DH params
* append to the cert file

* minor: adding .swp files to gitignore

### CODE REVIEW

File: rhui_provision/tasks/upgrade_ssl_configuration.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+- name: Generate DH params for cert
+  command: openssl dhparam -out /etc/pki/cds/dh2048.pem 2048
+
+- name: Backup old ssl cert
+  command: cp /etc/pki/cds/cds-ssl-cert.crt /etc/pki/cds/cds-ssl-cert.crt.bak
+
+- name: Concate DH params to SSL cert
+  command: cat /etc/pki/cds/dh2048.pem >> /etc/pki/cds/cds-ssl-cert.crt
```
Comment (jwmatthews) [inline rhui_provision/tasks/upgrade_ssl_configuration.yml]: Do you any thoughts of how we could make this idempotent, so we could re-run this ansible code on the same server multiple times without appending the 2048 key to the cert multiple times?

**Comment by jwmatthews** [comment]
Looks good overall, just a thought about considering if we could make this idempotent.

File: rhui_provision/tasks/upgrade_ssl_configuration.yml (hunk 1 above)
Comment (shawn-hurley) [inline rhui_provision/tasks/upgrade_ssl_configuration.yml]: +1

Comment (shawn-hurley) [inline rhui_provision/tasks/upgrade_ssl_configuration.yml]: I think that the latest one should do the trick.

**Comment by jwmatthews** [comment]
ACK

**Pull request merged**